src/Application/Controller.cpp

```diff
@@ -668,6 +668,7 @@ void Controller::aspectFill(Layout::Size size)
     Layout::Size targetSize = { pageSize.width * scaleFactor,
                                 pageSize.height }; // make sure to see all the contents
     m_layout->layout(targetSize);
+    m_layout->layout(targetSize, true);
```

Comment:
Better if there is any comment for the purpose of the second layout.

Reply:
Added.